cmd/alerts/silence/add_silence.go

```diff
@@ -65,27 +67,71 @@ func AddSilence(cmd *addSilenceCmd) {
 	}
 
 }
+func AddAllSilence(clusterID, duration, comment, username, clustername string, kubeconfig *rest.Config, clientset *kubernetes.Clientset) {
+	alerts := fetchAllAlerts(clusterID, kubeconfig, clientset)
+	for _, alert := range alerts {
+		addCmd := []string{
+			"amtool",
+			"silence",
+			"add",
+			"alertname=" + alert.Labels.Alertname,
+			"--alertmanager.url=" + utils.LocalHostUrl,
+			"--duration=" + duration,
+			"--comment=" + comment,
+		}
+
+		output, err := utils.ExecInPod(kubeconfig, clientset, addCmd)
+		if err != nil {
+			log.Fatal("Exiting the program")
+			return
+		}
 
+		formattedOutput := strings.Replace(output, "\n", " ", -1)
+
+		fmt.Printf("Alert %s has been silenced with id \"%s\" for a duration of %s by user \"%s\" \n", alert.Labels.Alertname, formattedOutput, duration, username)
+	}
+}
+
+func fetchAllAlerts(clusterID string, kubeconfig *rest.Config, clientset *kubernetes.Clientset) []utils.Alert {
+	var fetchedAlerts []utils.Alert
+
+	listAlertCmd := []string{"amtool", "--alertmanager.url", utils.LocalHostUrl, "alert", "-o", "json"}
+	output, err := utils.ExecInPod(kubeconfig, clientset, listAlertCmd)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = json.Unmarshal([]byte(output), &fetchedAlerts)
+	if err != nil {
+		log.Fatal("Error in unmarshaling the alerts", err)
+	}
+
+	return fetchedAlerts
+}
+
+/*
```

Comment:
If we don't need to keep this commented out section around, I'd say delete it entirely to remove confusion.